tunnel/IpTunnel.h

```diff
@@ -36,11 +36,21 @@ struct IpTunnel_Connection
     /** The IPv4 address used for this connection or all zeros if none was assigned. */
     uint8_t connectionIp4[4];
 
-    /** The IPv6 netmask/prefix length, in bits. Defaults to 128 if none was assigned. */
-    uint8_t connectionIp6Prefix;
+    /** The IPv6 netmask/prefix length from server, in bits.
+     * Defaults to 128 if none was assigned. */
+    uint8_t connectionIp6RoutedPrefix;
```

Comment:
If it would default to /0 it would make only one tunnel work. With default to 128 it is fully specified address.

Current default network size prefix makes no sense (/32 and /128) as it means that it is the only NIC in the network.